jerry-core/jmem/jmem.h

```diff
@@ -40,6 +40,23 @@
  */
 #define JMEM_ALIGNMENT (1u << JMEM_ALIGNMENT_LOG)
 
+/**
+ * Mask for tag part in jmem_cpointer_tag_t
+ */
+#define JMEM_TAG_MASK 0x7u
+#define JMEM_FIRST_TAG_MASK 0x1u
+#define JMEM_SECOND_TAG_MASK 0x2u
+#define JMEM_THIRD_TAG_MASK 0x4u
```

Comment:
In JerryScript you need a comment for each declaration. You can also use enums:
https://github.com/jerryscript-project/jerryscript/blob/master/jerry-core/ecma/base/ecma-globals.h#L66